src/TemplateLibrary/index.js

```diff
@@ -134,19 +132,18 @@ const TemplateLibraryComponent = (props) => {
           </HeaderImports>
         </Header>
         <Functionality>
-          <SearchInput className="icon" fluid icon="search" placeholder="Search..." onChange={onQueryChange} searchColor={libraryProps.SEARCH_COLOR} />
+          <SearchInput className="icon" fluid icon="search" placeholder="Search..." onChange={onQueryChange} className='templateLibrarySearchInput' />
           {props.addTemp
           && <NewClauseComponent addTempInput={props.addTemp} />}
         </Functionality>
-        {filtered && filtered.length ? <TemplateCards tempsHeight={libraryProps.TEMPLATES_HEIGHT} >
+        {filtered && filtered.length ? <TemplateCards className={'templateCardsDiv'} >
           {_.sortBy(filtered, ['name']).map(t => (
             <TemplateCard
               key={t.uri}
               addToCont={props.addToCont}
               template={t}
               handleViewTemplate={props.handleViewTemplate}
               className="templateCard"
```

Comment:
I believe this `className="templateCard"` can be removed, it's redundant on the `className='templateLibraryCard'` in `src/TemplateLibrary/Components/TemplateCard.js`